airbyte-tests/src/automaticMigrationAcceptanceTest/java/io.airbyte.test.automaticMigrationAcceptance/ImportApi.java

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
just a note the path for this file looks weird. i think there's one directory called "io.airbyte.test.automaticMigrationAcceptance" as opposed to "io/airbyte/test/automaticMigrationAcceptance"